src/figuresClasses.ts

```diff
@@ -1,19 +1,92 @@
-export interface Figure {
+export type FigureColor = 'red' | 'green' | 'blue';
+export type FigureShape = 'triangle' | 'circle' | 'rectangle';
 
+export interface Figure {
+  shape: FigureShape;
+  color: FigureColor;
+  getArea(): number;
 }
 
-export class Triangle {
+export class Triangle implements Figure {
+  shape: FigureShape = 'triangle';
 
-}
+  color: FigureColor;
+
+  a: number;
+
+  b: number;
 
-export class Circle {
+  c: number;
 
+  constructor(col: FigureColor, a: number, b: number, c: number) {
```

Comment:
Agree with Dasha, and code with the public is clearer.
And what the name is 'col' in your constructor?
```suggestion
  constructor(
    public color: FigureColor, 
    public a: number, 
    public b: number, 
    public c: number) {
```